x-pack/plugins/cloud_security_posture/public/pages/configurations/latest_findings_by_resource/resource_findings/resource_findings_container.test.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import { TestProvider } from '../../../../test/test_provider';
+import { useResourceFindings } from './use_resource_findings';
+import { FindingsBaseProps } from '../../../../common/types';
+import { ResourceFindings } from './resource_findings_container';
+
+jest.mock('./use_resource_findings', () => ({
+  useResourceFindings: jest.fn().mockReturnValue({
+    data: undefined,
+    error: false,
+  }),
+}));
+
+describe('<ResourceFindings />', () => {
+  it('should fetch resources with the correct parameters', async () => {
+    const props: FindingsBaseProps = {
+      dataView: {} as any,
+    };
+
+    render(
+      <TestProvider>
+        <ResourceFindings {...props} />
+      </TestProvider>
+    );
+
+    expect(useResourceFindings).toHaveBeenNthCalledWith(1, {
```

Comment:
this test doesn't verify the actual bug that happened, it would be nice to add another test case to check query is being passed